### ISSUE TITLE
Interfering

### ISSUE THREAD

**Issue by byronbonkers** [Shell1010/Selfcord#64]
PS D:\Coding\Discord Bots\AccBots> python '.\Acc Grabber.py'
Traceback (most recent call last):
  File ".\Acc Grabber.py", line 1, in <module>
    import selfcord
  File "C:\Users\byron\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selfcord\__init__.py", line 2, in <module>
    from .api import *
  File "C:\Users\byron\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selfcord\api\__init__.py", line 2, in <module>
    from .gateway import Activity, gateway
  File "C:\Users\byron\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selfcord\api\gateway.py", line 13, in <module>
    from selfcord.models.client import Client
  File "C:\Users\byron\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selfcord\models\__init__.py", line 2, in <module>
    from .channel import (Category, DMChannel, GroupChannel, Messageable,
  File "C:\Users\byron\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selfcord\models\channel.py", line 15, in <module>
    from ..utils import logging
  File "C:\Users\byron\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selfcord\utils\__init__.py", line 2, in <module>
    from .command import (Command, CommandCollection, Context, Event, Extender,
  File "C:\Users\byron\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selfcord\utils\command.py", line 8, in <module>
    from typing import TYPE_CHECKING, Any, get_origin
ImportError: cannot import name 'get_origin' from 'typing' (C:\Users\byron\AppData\Local\Programs\Python\Python37-32\lib\typing.py)
PS D:\Coding\Discord Bots\AccBots> 
```
import selfcord

token = "MY TOKEN"
bot = selfcord.Bot()

@bot.on("ready")
async def ready(time):
    print(f"Connected To {bot.user.name}\n Startup took {time:0.2f} seconds")

@bot.on("message")
async def responder(message):
    if message.content == "ping!":
        await message.channel.send("pong!")

bot.run(token)
```

**Comment by Shell1010** [comment]
OK not sure how that managed to get in there... My bad I'll push a hotfix.

**Comment by Shell1010** [comment]
Solved in recent release #65
